test/binet-test.js

```diff
@@ -52,4 +52,57 @@ describe('binet', function() {
     assert.strictEqual(binet.encode(raw4), ip4);
     assert.strictEqual(binet.encode(raw6), ip6);
   });
+
+  it('should return the correct network', () => {
+    assert.strictEqual(
+      binet.getNetwork(binet.decode('127.0.0.1')),
+      binet.networks.NONE
+    );
+
+    assert.strictEqual(
+      binet.getNetwork(binet.decode('::1')),
+      binet.networks.NONE
+    );
+
+    assert.strictEqual(
+      binet.getNetwork(binet.decode('8.8.8.8')),
+      binet.networks.INET4
+    );
+
+    assert.strictEqual(
+      binet.getNetwork(binet.decode('2001::8888')),
```

Comment:
Currently Binet is returning the Onion network for this and not INET6. Investigating...

Reply:
Fix w/ force push, this is correctly returning Teredo, Bitcoin only tests for that in their NetworkExt property, but I this is better.